indexer-app/src/cmr/indexer/data/concepts/collection.clj

```diff
@@ -211,11 +211,11 @@
     (let [free-and-open (:FreeAndOpenData use-constraints)]
       (case free-and-open
         true (distinct (conj consortiums "GEOSS"))
-        false (remove #(= "GEOSS" (str/upper-case %)) consortiums)
+        false (remove #(= "GEOSS" %) consortiums)
```

Comment:
why do we want to make any code case sensitive? I see the change on line 232, but do we ever expect to have a 'geoss' that is different from 'GEOSS'?

Reply:
It's not case sensitive. consortiums is already upper-cased in the index.